.github/workflows/linter.yml

```diff
@@ -17,7 +17,7 @@ jobs:
     runs-on: blacksmith-4vcpu-ubuntu-2204
     strategy:
       matrix:
-        python-version: ["3.12"]
+        python-version: ["3.13"]
```

Comment:
Does the `lint` job step pass if this change is reverted?

Reply:
Yes

Reply:
But I think tox.ini needs to set 3.12 too if we want to revert it for the lint.

Reply:
We can leave in on Py3.13...  We will need to fix it at some point so let's do it now.